docs/application/dotnet/guides/nui/text.md

```diff
@@ -417,7 +449,30 @@ The following example illustrates text alignment:
 field.HorizontalAlignment = HorizontalAlignment.Begin;
 ```
 
-### Use Input Properties in TextField
+### Use HiddenInputSettings property in TextField
+
+The `TextField` class can substitute the input text using the `HiddenInputSettings` property.
+
+To set the `HiddenInputSettings` for detail options, you can use [HiddenInputProperty](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.HiddenInputProperty.html) and [HiddenInputModeType](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.HiddenInputModeType.html).

```

Comment:
Just one doubt :  To set the `HiddenInputSettings` for detailed options, 
What are the detailed options? 

I am not able to get this, can you elaborate. 
```suggestion
To set the `HiddenInputSettings` for detailed options, you can use [HiddenInputProperty](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.HiddenInputProperty.html) and [HiddenInputModeType](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.HiddenInputModeType.html).

Reply:
@nawab-samsung 

The detailed options are Mode and Properties of HiddenInputSettings.
(Struct HiddenInputProperty, Enum HiddenInputModeType)

Depending on HiddenInputModeType, TextField behaves as follows.
* HideAll : Hide all the input text.
* HideCount : Hide n characters from start.
* HideNone : Don't hide text.
* ShowCount : Show n characters from start.
* ShowLastCharacter : Show last character for the duration(use ShowLastCharacterDuration property to modify duration).

Also, depending on the mode, through HiddenInputProperty,
ShowLastCharacterDuration, SubstituteCharacter, SubstituteCount values ​​can be set.

I think, users need a detailed explanation of this.
So I made an API reference link for this. (The most detailed explanation)

Reply:
@wonrst thanks.